scripts/build_and_verify_py_packages.sh

```diff
@@ -5,7 +5,7 @@
 # LICENSE file in the root directory of this source tree.
 
 # Get version number (created dynamically via setuptools-scm)
-version=$(python setup.py --version)
+version=$(python -m setuptools_scm)
```

Comment:
Is it possible to error out and exit if this fails? The default behavior of the bash script is to print out the error message (i.e., `No module named setuptools_scm`) and keep executing the rest of the script. I think things will fail either way, but it would make it easier to pinpoint the root cause of the failure.

Reply:
Sure, we could just regex our way around this and see if `version` is a `No module...` string. Will do that.